solidity/dashboard/src/utils/ethereum.utils.js

```diff
@@ -43,3 +46,32 @@ export const getEventsFromTransaction = async (contractToEventName, txHash) => {
 
   return events
 }
+
+// We want to prevent long values being displayed in ETH unit(eg.
+// 0.000034560123345621 ETH). So in that case we want to display `<0.0001`.
+// More info here:
+// https://github.com/keep-network/keep-core/pull/2050#issuecomment-693434991
+export const MIN_ETH_AMOUNT_TO_DISPLAY_IN_WEI = toBN(10)
+  .pow(toBN(14))
+  .toString() // 0.0001 ETH
+
+export function displayEthAmount(
+  amountInWei,
+  unit = "ether",
+  decimalsPlaces = 4
+) {
+  if (!amountInWei) {
+    return 0
```

Comment:
Let's return `"0"` as a string so this function consistently returns a string, and check for `amountInWei == "0"` or perhaps `parseInt(amountInWei) == 0`. Right now a 0 reward shows `<0.0001`.

Reply:
pls see 51a8003